packages/esm-patient-labs-app/src/lab-orders/lab-order-basket-panel/lab-order-basket-panel.scss

```diff
@@ -16,21 +16,14 @@
   margin-bottom: spacing.$spacing-03;
 }
 
-.desktopHeading {
+.desktopHeading, .tabletHeading {
```

Comment:
I think if this requires re-styling the background, we should just collapse this class into `.heading` and apply it unconditionally.